clients/ptranslator/src/test/java/org/hawkular/metrics/clients/ptrans/fullstack/CollectdITest.java

```diff
@@ -95,7 +94,7 @@
     @Before
     public void setUp() throws Exception {
         tenant = getRandomTenantId();
-        findNumericMetricsUrl = "http://" + BASE_URI + "/" + tenant + "/metrics?type=num";
+        findGuageMetricsUrl = "http://" + BASE_URI + "/" + tenant + "/metrics?type=guage";
```

Comment:
Typo "guage" (is this the cause of Travis failures?) Note that there are other places in this file and in other files with the same typo.